stream-chat-android-client/src/androidTest/java/io/getstream/chat/android/client/utils/TestChatUtils.kt

```diff
@@ -1,20 +0,0 @@
-package io.getstream.chat.android.client.utils
-
-import org.assertj.core.api.Assertions.assertThat
-import org.assertj.core.api.Assertions.assertThatThrownBy
-import org.junit.Test
-
-internal class TestChatUtils {
```

Comment:
We have tests for `devToken` elsewhere (plus it's not even in `ChatUtils` anymore, that class doesn't exist now)